script.go

```diff
@@ -0,0 +1,77 @@
+package rockscache
+
+import "github.com/redis/go-redis/v9"
+
+var (
+	deleteScript = redis.NewScript(`redis.call('HSET', KEYS[1], 'lockUntil', 0)
```

Comment:
Can you add a new line at the first of the script? That may make the code more readable

Reply:
done